pkg/common/cns-lib/volume/manager.go

```diff
@@ -1640,7 +1644,11 @@ func (m *defaultManager) expandVolumeWithImprovedIdempotency(ctx context.Context
 
 		clusterFlavor = expandVolParams.ClusterFlavor
 		if expandVolParams.IsPodVMOnStretchSupervisorFSSEnabled && clusterFlavor == cnstypes.CnsClusterFlavorWorkload {
-			reservedQty := resource.NewQuantity(size, resource.BinarySI)
+			// for expand volume, reserved need to set to the difference between the new volume size and current volume capacity
+			// param "size" is the sizeInMb, need to convert it to sizeInBytes
+			sizeInBytes := size * MbInBytes
```

Comment:
`size` is current size of volume? How do you get it?
and `expandVolParams.Capacity` is the new size?

Reply:
I think `size` is the new size passed in from the CSI expand volume call and `expandVolParams.Capacity` is the original size which is from CNSVolumeInfo.
@lipingxue can you add some comments to clarify?

Reply:
Fixed.